test/lib.bash

```diff
@@ -203,6 +203,7 @@ function run_rolling_upgrade_tests {
   E2E_UPGRADE_TESTS_SERVING_USE=true \
   E2E_UPGRADE_TESTS_CONFIGMOUNTPOINT=/.config/wathola \
   E2E_UPGRADE_TESTS_INTERVAL="50ms" \
```

Comment:
What do you think about lowering this interval (between events)? EventingKafka on upstream operates with `2 msec`

https://github.com/knative-sandbox/eventing-kafka/blob/48351009c15aa61fa0837b7fc3ac6aba286c6ad9/test/upgrade/continual.go#L48

Reply:
IMO, 50ms (or 20 times per second) is a good enough setting for upgrade tests. Should we implement a load test we'd need to increase it. But for this case it looks good to me.